src/core.js

```diff
@@ -210,6 +233,35 @@ export default class Core extends Emitter {
    * Do some tests before HTTP requests to optimize pipeline.
    */
   async beforeFetch() {
+    const urlBeforeHistoryPush = window.location.href;
+
+    this.emit('BEFORE_HISTORY', {
+      from: {
+        page: this.From.properties.page,
+        view: this.From.properties.view
+      },
+      trigger: this.trigger,
+      location: this.location
+    });
+
+    let goToSleep = false;
+    let fetchPage = true;
+
+    if (urlBeforeHistoryPush === this.asleep.href) {
+      fetchPage = false;
+    }
```

Comment:
Deze snap ik niet helemaal, je checkt hier of de huidige URL gelijk is aan de url van de slapende pagina, voor m'n gevoel zou je juist moeten checken of de huidige URL gelijk is aan de actieve pagina (die 'over' de slapende heen ligt), waarom is dat zo?